app/mm-redux/utils/emoji_utils.ts

```diff
@@ -17,11 +17,11 @@ export function getEmojiImageUrl(emoji: Emoji): string {
         return Client4.getEmojiRoute(emoji.id) + '/image';
     }
 
-    const filename = emoji.filename || emoji.aliases[0];
+    const filename = emoji.image || emoji.short_name;
```

Comment:
do we need this change? I guess `getEmojiImageUrl` is not needed and we only need the one for custom emoji's.

Also can we rename `isSystemEmoji` on line 7, do we need it ?